src/gluonts/model/rotbaum/_preprocess.py

```diff
@@ -404,12 +404,26 @@ def make_features(self, time_series: Dict, starting_index: int) -> List:
             feat_static_cat = []
 
         feat_dynamic_real = (
-            [elem for ent in time_series["feat_dynamic_real"] for elem in ent]
+            list(
+                chain(
+                    *[
+                        list(ent[0]) + list(ent[1].values())
```

Comment:
what is `ent`? can we use a more descriptive name?

Reply:
ent short for entry, presumably. I borrowed the same convention introduced elsewhere in the code. In this particular case, we probably can't be too descriptive: ent here is a tuple; the first coordinate is a transformed window, and the second one is a dictionary of additional features. If we wanted it to be more descriptive it would have to be upstream in the output format of _pre_transform.